server/api/rounds.py

```diff
@@ -519,6 +616,31 @@ def create_round(election: Election):
     return jsonify({"status": "ok"})
 
 
+@api.route("/election/<election_id>/round/<round_id>/finish", methods=["POST"])
```

Comment:
Is it ever the case that we need to finish a round other than the most current? Wondering if this endpoint really requires a `round_id` param

Reply:
Good q. No, we don't need to finish other rounds.

The convention in Arlo has been to overspecify params in the API urls and then verify them in the endpoint (e.g. check out the pattern in `undo_round_start`). So this endpoint should validate that the given round id is the current round (which I'm now realizing is missing here).

I'm feeling unsure whether it's better to change to the style you suggest (which I agree is simpler), or to stick to the more REST-y convention of specifying explicitly the resource you want to modify.

Reply:
Got it - I could go either way on this one. I do like consistency, but also think that it's okay to diverge from this convention if it isn't doing much for us

Reply:
Decided to update to `/election/<election_id>/rounds/current/finish` to still have a bit of that REST-y vibe will still getting the benefits of the simplification. And updated the undo round start endpoint to match. Thanks for the suggestion!

Reply:
Nice, I like that compromise!